lib/dialects/postgres/data-types.js

```diff
@@ -422,7 +422,10 @@ module.exports = BaseTypes => {
         this.toCastType();
     }
     const valuesStringified = values.map(value => {
-      if (this.options.subtype.stringify) {
+      if (value === null || value === -Infinity || value === Infinity) {
```

Comment:
use `includes` here

```js
if ([null, -Infinity, Infinity].includes(value)) {
```

May be Node 4 doesnt supports it, if it doesn't then https://lodash.com/docs/#includes